src/Evolution/VariableFixing/FixToAtmosphere.hpp

```diff
@@ -52,8 +53,16 @@ class FixToAtmosphere {
     static type lower_bound() { return 0.0; }
     static constexpr OptionString help = {"Density of atmosphere"};
   };
+  /// \brief Rest mass density at which to impose the atmosphere. Should be
+  /// greater than the density of the atmosphere.
+  struct DensityCutoff {
+    using type = double;
+    static type lower_bound() { return 0.0; }
+    static constexpr OptionString help = {
+        "Density to impose atmosphere at. Must be > rho_atm"};
+  };
 
-  using options = tmpl::list<DensityOfAtmosphere>;
+  using options = tmpl::list<DensityOfAtmosphere, DensityCutoff>;
   static constexpr OptionString help = {
```

Comment:
maybe rephrase this OptionString to say "If the rest mass density is below the cutoff value, it is raised to the atmosphere value" ?

Reply:
Will's comment below made me realize my suggestion is wrong --- "..., it is set to the atmosphere value" is correct.